public/javascripts/SVValidate/src/Main.js

```diff
@@ -209,6 +209,486 @@ function Main (param) {
+                        'slideSubtitle': '',
+                        'slideDescription': i18next.t('mission-tutorial.signal.slide-1.description'),
+                        'imageURL': 'assets/images/tutorials/signal-correct-1.png',
+                        'onImageLabel': {
+                            'title': i18next.t('mission-tutorial.on-image-label-title-correct'),
+                            'description': i18next.t('mission-tutorial.on-image-label-description-correct'),
+                            'position' : {
+                                'left': '167px',
+                                'top': '325px'
+                            }
+                        }
+                    },
+                    {
+                        'isExampleCorrect': false,
+                        'exampleTypeLabel': i18next.t('mission-tutorial.example-type-label-incorrect'),
+                        'exampleTypeIcon': '#smile-negative',
+                        'slideTitle': i18next.t('mission-tutorial.signal.slide-2.title'),
+                        'slideSubtitle': i18next.t('mission-tutorial.label-type-subtitle'),
+                        'slideDescription': i18next.t('mission-tutorial.signal.slide-2.description'),
+                        'imageURL': 'assets/images/tutorials/signal-incorrect-1.png',
+                        'onImageLabel': {
+                            'title': i18next.t('mission-tutorial.on-image-label-title-incorrect'),
+                            'description': i18next.t('mission-tutorial.on-image-label-description-incorrect'),
+                            'position' : {
+                                'left': '377px',
+                                'top': '86px'
+                            }
+                        }
+                    },
+                    {
+                        'isExampleCorrect': false,
+                        'exampleTypeLabel': i18next.t('mission-tutorial.example-type-label-incorrect'),
+                        'exampleTypeIcon': '#smile-negative',
+                        'slideTitle': i18next.t('mission-tutorial.signal.slide-3.title'),
+                        'slideSubtitle': i18next.t('mission-tutorial.label-type-subtitle'),
+                        'slideDescription': i18next.t('mission-tutorial.signal.slide-3.description'),
+                        'imageURL': 'assets/images/tutorials/signal-incorrect-2.png',
+                        'onImageLabel': {
+                            'title': i18next.t('mission-tutorial.on-image-label-title-incorrect'),
+                            'description': i18next.t('mission-tutorial.on-image-label-description-incorrect'),
+                            'position' : {
+                                'left': '390px',
+                                'top': '203px'
+                            }
+                        }
+                    },
+                    {
+                        'isExampleCorrect': false,
```

Comment:
What are your thoughts on using the value of `isExampleCorrect` to infer the correct values for `exampleTypeLabel`, `exampleTypeIcon`, `onImageLabel.title`, and `onImageLabel.description` in `MissionTutorial.js` instead of needing to define them in the descriptor descriptor for each slide?

With 25 slides, that'd remove 100 lines from the descriptor definition!

Reply:
Sure we can do that! Customization capability can be added later if required.